cwltest/__init__.py

```diff
@@ -307,6 +307,7 @@ def main():  # type: () -> int
             for i, job in zip(ntest, jobs):
                 test_result = job.result()
                 test_case = test_result.create_test_case(tests[i])
+                test_case.classname = "Test {}".format(i + 1)
```

Comment:
I don't think `classname` is the right attribute; we already use that via `args.classname` / `cwltest --classname python3` (for example); so this would overwrite that.

Reply:
@mr-c Sorry I missed the inclusion of that feature. Another contender would be the `url` attribute; we could make it a legitimate URL by using a custom scheme like `cwltest:v1.0/conformance_test_v1.0.yaml#42`. The added line would then become:

`test_case.url = "cwltest:{}#{}".format(args.test, i + 1)`

 Do you agree with this approach?

Reply:
@mr-c can we merge this?